### ISSUE TITLE
HTML preview is very difficult to read in the dark theme

### ISSUE THREAD

**Issue by cady-b** [typst/webapp-issues#905]
<img width="631" height="213" alt="Image" src="https://github.com/user-attachments/assets/f7e19cb3-0734-44df-8045-6d9523ad539e" />

With a contrast ratio of 1.19:1[^viawebaim], black on `#19181F` [fails WCAG AA](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html) by a huge margin (difference in HSL lightness to the next passing value covers about 65% of the whole range).

[^viawebaim]: via https://webaim.org/resources/contrastchecker

The Invert Preview option also doesn't work in HTML, so I'm stuck either toggling the theme as a whole or manually adding a stylesheet.

I think it would be nicest to have the iframe's background be white; that'd match paged export and how browsers render the default background, while users can still overwrite it with a stylesheet just as before. If changing this by default is not desired, making it an option in the View menu or fixing Invert Preview could also resolve the immediate problem.

Alternatively text could be made white by default as in the editor, but that might lead to surprising results with the exported page looking different.
